packages/manager-components/src/components/molecules/links/linkWithArrow/linkWithArrow.component.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsLink, OsdsIcon } from '@ovhcloud/ods-components/react';
+import {
+  OdsHTMLAnchorElementRel,
+  OdsHTMLAnchorElementTarget,
+} from '@ovhcloud/ods-common-core';
+
+export enum LinkType {
+  back = 'back',
+  next = 'next',
+}
+
+export interface LinkWithArrowProps {
+  download?: string;
+  label?: string;
+  href?: string;
+  rel?: OdsHTMLAnchorElementRel;
+  target?: OdsHTMLAnchorElementTarget;
+  type?: LinkType;
+}
+
+export const LinkWithArrow: React.FC<LinkWithArrowProps> = ({
+  download,
+  label,
+  href,
+  target,
+  type,
+  rel,
+}: LinkWithArrowProps) => {
+  const arrowIcon =
+    type === LinkType.back
+      ? ODS_ICON_NAME.ARROW_LEFT
+      : ODS_ICON_NAME.ARROW_RIGHT;
+  const arrowClass = type === LinkType.back ? 'pr-5' : 'pl-5';
+
+  return (
+    <div className="mb-5">
+      {type === LinkType.back && (
+        <span slot="start">
+          <OsdsIcon
+            className={`align-middle ${arrowClass}`}
+            name={arrowIcon}
+            size={ODS_ICON_SIZE.xxs}
+            color={ODS_THEME_COLOR_INTENT.primary}
+          />
+        </span>
+      )}
+      <OsdsLink
+        color={ODS_THEME_COLOR_INTENT.primary}
+        download={download}
+        href={href}
+        target={target}
+        rel={rel}
+        data-testid="link-arrow-id"
```

Comment:
Don't keep test props in source code
Spread additional props in the component instead

```
const Example = ({prop1, prop2, ...props}: {prop1: string; prop2: string} & Record<string, string>) => (
  <div prop1={prop1} prop2={prop2} {...props} />
)
```